cypress/integration/subscription_modal.ts

```diff
@@ -67,7 +67,7 @@ const testSharedSubscriptionModalFunctionality = (
       it("has an invalid slack username", () => {
         selectAntdOption("notification-method-select", "Slack");
         cy.dataCy("slack-input").clear();
-        cy.dataCy("slack-input").type("sart");
+        cy.dataCy("slack-input").type("sa rt");
```

Comment:
I think it would make sense if `sart` continued to fail the regex, considering the member ID is all caps (plus numbers) and we even indicate that in the hint. I think one regex that would work is `/(#|@).+|(^[A-Z0-9]+$)/`, not sure if it can be simplified further.

Reply:
I know, I also wish we could narrow it down more. The problem is that while they seem to be all caps right now, slack seems to be suggesting against relying on it, as they provide no guarantees for the format. 

 "We recommend considering the format of the string to be an opaque value, and not to rely on a particular structure."